src/events/deposits.upsert.service.ts

```diff
@@ -114,33 +123,51 @@ export class DepositsUpsertService {
     blockTimestamp: Date | null,
   ): Promise<void> {
     if (!deposit.main) {
-      const event = await prisma.event.findUnique({
-        where: {
-          deposit_id: deposit.id,
+      const event = await tracer.trace(
+        'DepositUpserService.processDeposit.find_event',
+        async () => {
+          return prisma.event.findUnique({
+            where: {
+              deposit_id: deposit.id,
+            },
+          });
         },
-      });
+      );
+
       if (event) {
-        await this.eventsService.deleteWithClient(event, prisma);
+        await tracer.trace(
+          'DepositUpsertService.processDeposit.delete_event',
```

Comment:
nit: It's reads a little weird to have mixed camelCase and snake_case

Reply:
it does look odd! since these names are only for our benefit I'll stick to camelCase